components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,25 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  NSAssert(thumbRadius >= 0, @"Thumb radius must be a positive value");
+  float radius = (float)thumbRadius;
+  _thumbRadiusesForState[@(state)] = [NSNumber numberWithFloat:radius];
+  if (self.state == state) {
```

Comment:
Please update the thumb radius regardless of the current state. Consider what happens if it is currently `.highlighted` and the value for `.normal` was just updated. If we're relying on the fallback behavior for `.highlighted`, then we should see the change immediately.

Reply:
Done